package/boot/uboot-mediatek/patches/420-add-support-for-RAVPower-RP-WD009.patch

```diff
@@ -140,6 +140,7 @@ Subject: [PATCH] add support for RAVPower RP-WD009
 +CONFIG_SYS_CONSOLE_INFO_QUIET=y
 +CONFIG_VERSION_VARIABLE=y
 +CONFIG_BOARD_RAVPOWER_RP_WD009=y
++CONFIG_SYS_MIPS_TIMER_FREQ=290000000
```

Comment:
not documented change?

Reply:
Because this value is not missing the .config, the make config run hangs and prompts for the value. 
This prompt is about 1Mio times repeated in the log output of the runner, until it timeouts. 
`Build U-Boot image with exception vectors (ROM_EXCEPTION_VECTORS) [Y/?] y
Fixed MIPS CPU timer frequency in Hz (SYS_MIPS_TIMER_FREQ) [] (NEW) 
`
The value comes from the include file of the same target. 
I'll expand the documentation